src/Stratis.Features.FederatedPeg/FederatedPegFeature.cs

```diff
@@ -152,6 +156,9 @@ public override async Task InitializeAsync()
             // Query our database for partially-signed transactions and send them around to be signed every N seconds.
             this.partialTransactionRequester.Start();
 
+            // Cleans the mempool.
+            this.mempoolCleaner.Start();
```

Comment:
To my knowledge, the mempool re-evaluates all added entries on startup. Any entries that are invalid or already appear in blocks should get removed during that process.

Reply:
Can change the comment to `// Cleans the mempool periodically.` ?

Reply:
Yeah, I think that would make it more clear what the intention of the cleaner is.

Reply:
Done.